org-brain.el

```diff
@@ -418,7 +506,8 @@ the concept map buffer will gain focus."
           (max-width 0))
       (mapc (lambda (parent)
               (push parent ignored-siblings)
-              (let ((children-links (-difference (org-brain-children parent entry)
+              (let ((children-links (set-difference
```

Comment:
All `cl` functions should use their `cl-` name, so `cl-set-difference` in this case.